server/controller/index.js

```diff
@@ -0,0 +1,27 @@
+const Question = require('../models/question.js');
+
+exports.addQuestion = async (ctx) => {
+  const question = ctx.request.body;
+
+  const newQuestion = new Question(question);
+
+  newQuestion.save((err) => {
+    if (err) {
+      console.error(err, 'error');
```

Comment:
The second argument is pointless as it's already an error message and should show the error, `console.error(err);` should be enough.